src/language-css/index.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const printer = require("./printer-postcss");
+const { lineColumnToIndex, getLast } = require("../common/util");
```

Comment:
Node 4 doesn’t like destructuring :(

Reply:
Oops - fixed.

Reply:
Node 4 is expiring soon :) There was a plan to drop it at one point..

Reply:
That would/will be nice 😃

Reply:
@lipis:

> Looks good except for node v4 compat. Was chatting with **@​vjeux** and we think we shouldn't force people to upgrade versions, especially since Prettier is often run in CI.

https://github.com/prettier/prettier/issues/3503#issuecomment-359937346

Reply:
I will just close that one then: #3919